tests/core/test_core_meds.py

```diff
@@ -0,0 +1,180 @@
+"""Tests for core__medicationrequest"""
+
+import json
+
+import pytest
+
+from tests import testbed_utils
+
+RXNORM = "http://www.nlm.nih.gov/research/umls/rxnorm"
+
+
+@pytest.mark.parametrize(
+    "dosage,expected",
+    [
+        (None, [None]),  # No dosage
+        ([{"sequence": 12}], [None]),  # Irrelevant dosage
+        ([{"text": "One"}], ["One"]),  # Single dosage
+        (  # Multiple dosages
+            [{"text": "Multi1"}, {"text": "Multi2"}, {"sequence": 1}],
+            ["Multi1", "Multi2"],
+        ),
+    ],
+)
+def test_core_medreq_dosage(tmp_path, dosage, expected):
+    """Verify that dosage text is optionally included"""
+    testbed = testbed_utils.LocalTestbed(tmp_path)
+    testbed.add_medication_request("A", dosageInstruction=dosage)
+    con = testbed.build()
+    texts = con.sql(
+        "SELECT dosageInstruction_text FROM core__medicationrequest "
+        "ORDER BY dosageInstruction_text"
+    ).fetchall()
+    assert [x[0] for x in texts] == expected
+
+
+@pytest.mark.parametrize(
+    "codings,expected",
+    [
+        ([{"code": "A", "system": RXNORM}], ["A"]),  # one code
+        ([{"code": "A", "system": "nope"}], []),  # skip non-rxnorm
+        (
+            [{"code": "A", "system": RXNORM}, {"code": "B", "system": "nope"}],
+            ["A"],  # ignores non-rxnorm, but keeps good ones
+        ),
+        (
+            [{"code": "A", "system": RXNORM}, {"code": "B", "system": RXNORM}],
+            ["A", "B"],
+        ),
+    ],
+)
+def test_core_medreq_only_rxnorm(tmp_path, codings, expected):
+    """Verify that we only include rxnorm codes
+
+    Note: this test was written against the found behavior at the time.
+    It's not clear this is how we *want* this table to behave.
```

Comment:
Callout for this test and the one below - I'm encoding current behavior in these tests, but it's not clear to me what our vision for these tables is. I filed #227 for that discussion.